src/components/BestItemBanner.js

```diff
@@ -0,0 +1,32 @@
+import { useState, useEffect } from "react";
+import { getItems } from "../api";
+import ItemBox from "./ItemBox";
+import "../styles/BestItemBanner.css";
+
+const BestItemBanner = () => {
+  const [bestItem, setBestItem] = useState([]);
```

Comment:
list 형태는 복수형의 이름을 사용하면 좋아요 
```suggestion
  const [bestItems, setBestItems] = useState([]);
```